app.py

```diff
@@ -0,0 +1,26 @@
+from flask import Flask, render_template, request
+import sys
+
+sys.path.insert(0, 'src')
```

Comment:
This is probably me not knowing python, but is this idiomatic? I see you use it in your tests too. What does this line achieve, and is there a different way to do it?